Code/Editor/AzAssetBrowser/AzAssetBrowserRequestHandler.cpp

```diff
@@ -412,8 +413,22 @@ void AzAssetBrowserRequestHandler::AddContextMenuActions(QWidget* caller, QMenu*
 {
     using namespace AzToolsFramework::AssetBrowser;
 
-    AssetBrowserTreeView* treeView = qobject_cast<AssetBrowserTreeView*> (caller);
-    if (!treeView)
+    QString callerName = QString();
+    bool calledFromAssetBrowser = false;
+
+    AssetBrowserTreeView* treeView = qobject_cast<AssetBrowserTreeView*>(caller);
+    if (treeView)
+    {
+        calledFromAssetBrowser = treeView->GetIsAssetBrowserMainView();
+    }
+
+    AssetBrowserTableView* tableView = qobject_cast<AssetBrowserTableView*>(caller);
+    if (tableView)
+    {
+        calledFromAssetBrowser |= tableView->GetIsAssetBrowserMainView();
```

Comment:
Just out of curiosity, why did you decide to use a bitwise or assignment instead of an `=` here?

Reply:
I was being over cautious: there should never be an instance of treeView being set at the same time as tableView, so the or is irrelevant really.